packages/http/src/mocker/HttpMocker.ts

```diff
@@ -38,16 +38,10 @@ export class HttpMocker
       try {
         negotiationResult = helpers.negotiateOptionsForInvalidRequest(resource.responses);
       } catch (error) {
-        return {
-          statusCode: 400,
-          headers: {
-            'Content-type': 'text/plain',
-          },
-          body: `ERROR: Your request is not valid.
-We cannot generate a sensible response because your '400'
-response has neither example nor schema or is not defined.
-Here is the original validation result instead: ${JSON.stringify(input.validations.input)}`,
-        };
+        throw ProblemJsonError.fromTemplate(
```

Comment:
I'm trying to think if we could be more specific here? E.g. we don't check what the `error` is.

For instance, the original error message says:

> We cannot generate a sensible response because your '400'	
> response has neither example nor schema or is not defined.	
> Here is the original validation result instead: ${JSON.stringify(input.validations.input)}`,

Which leads me to think that there is more than one failure scenario. Maybe the original message didn't make sense but it looks like it's two different things: invalid request body VS undefined example/schema. Have you been able to verify which one it actually is?

Reply:
The original error was more specific, @philsturgeon asked to be generic because we might want to change the way Prism responds.

Ideally, instead of saying "Hey I can't find an error response for this invalid request", we could in the future generate such error response automatically.